distribution/pull_v2_test.go

```diff
@@ -295,7 +295,7 @@ func TestPullSchema2Config(t *testing.T) {
 				switch {
 				case r.Method == "GET" && r.URL.Path == "/v2":
 					w.WriteHeader(http.StatusOK)
-				case r.Method == "GET" && r.URL.Path == "/v2/docker.io/library/testremotename/blobs/"+expectedDigest.String():
+				case r.Method == "GET" && r.URL.Path == "/v2/library/testremotename/blobs/"+expectedDigest.String():
```

Comment:
As far as I could tell, this `docker.io/` in the path was purely to match the URLs we constructed in the test (which didn't set `TrimHostpath`, but in non-test code, it would be set?